### PR TITLE
cdktf 0.0.16 (new formula)

### PR DESCRIPTION
**Pull request by theabrar**
CDK for Terraform
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/cdktf.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  depends_on "terraform"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    assert_match "Cannot initialize a project in a non-empty directory",
+      shell_output("#{bin}/cdktf init python-app 2>&1", 1)
```
Comment (anubhavmishra) [inline Formula/cdktf.rb:20]:
```suggestion
      shell_output("#{bin}/cdktf init python-app 2>&1", 1)
```
```suggestion
      shell_output("#{bin}/cdktf init --template='typescript' 2>&1", 1)
```

File: Formula/cdktf.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  depends_on "node"
+  depends_on "terraform"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    assert_match "Cannot initialize a project in a non-empty directory",
```
Comment (anubhavmishra) [inline Formula/cdktf.rb:19]:
```suggestion
    assert_match "ERROR: Cannot initialize a project in a non-empty directory",
```

Comment (theabrar) [reply Formula/cdktf.rb]: done.

File: Formula/cdktf.rb (hunk 1 above)
Comment (theabrar) [reply Formula/cdktf.rb]: I used --template='python' instead, as this is primarily for non npm/typescript users

**Review by chenrui333 — APPROVED** [review]
lgtm

**Comment by chenrui333** [comment]
@theabrar, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**